vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -568,7 +568,11 @@ def allowed_templates(options = {})
       _log.warn "Allowed Templates is returning an empty list"
     else
       _log.warn "Allowed Templates is returning <#{allowed_templates_list.length}> template(s)"
-      allowed_templates_list.each { |vm| _log.info "Allowed Template <#{vm.id}:#{vm.name}>  GUID: <#{vm.guid}>  UID_EMS: <#{vm.uid_ems}>" }
+      if _log.debug?
+        allowed_templates_list.each do |vm|
+          _log.info "Allowed Template <#{vm.id}:#{vm.name}>  GUID: <#{vm.guid}>  UID_EMS: <#{vm.uid_ems}>"
```

Comment:
I'll carefully avoid asking which part of this is expensive.

But if we're only logging it when we're at debug level, let's write it at debug level. To do otherwise seems likely to be Very Confusing Indeed.